app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodFacade
+  def self.results(query)
+    parsed_json = FoodService.search(query)
```

Comment:
Can you think of a way you could use a single method to get all the info that you want?